### DISCUSSION TITLE
Open_pdks discussions

### DISCUSSION THREAD

**Discussion by RTimothyEdwards** [fossi-foundation/open-pdks#208] [Announcements]
## Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MOS5500** [comment]
Hi to all!
My name is Claudio, I'm an Italian software developer and Computer Scienze teacher. I'm a computer architecture passionate too.
I'm trying to learn how to use Open Pdk because I would like to build a CPU on silicon with a new type of architecture that we have designed and currently there is a prototype based on FPGA.
